endpoint.rst

```diff
@@ -8,10 +8,10 @@ Overview
 
 The ``remote`` command group allows users to manage the service endpoints
 {Singularity} will interact with for many common command flows. This includes
-managing credentials for image storage services, remote builders, and key 
+managing credentials for image storage services, remote builders, and key
 servers used to locate public keys for SIF image verification. Currently,
 there are three main types of remote endpoints managed by this command group:
-the public Sylabs Cloud (or local {Singularity} Enterprise installation), OCI 
+the public Sylabs Cloud (or local {Singularity} Enterprise installation), OCI
```

Comment:
```suggestion
the public Sylabs Cloud (or local Singularity Enterprise installation), OCI
```

If I'm understanding the reason for {} here, this should be left as "Singularity Enterprise"